### PR TITLE
feat: Add aria-label prop to Loading component

### PR DESCRIPTION
**Pull request by frankieyan**
## Short description

Adds an optional `aria-label` prop to `<Loading>`

## PR Checklist

-   [x] Added tests for bugs / new features
-   [x] Updated docs (storybooks, readme)
-   [x] Executed `npm run validate` and made sure no errors / warnings were shown
-   [x] Described changes in `CHANGELOG.md`
-   [x] Bumped version in `package.json` and `package-lock.json` (`npm --no-git-tag-version version <major|minor|patch>`) [ref](https://docs.npmjs.com/cli/v6/commands/npm-version)
-   [x] Updated all static build artifacts (`npm run build-all`)

## Versioning

v9.1.0

### CODE REVIEW

**Review by craigcarlyle — COMMENTED** [review]
Nice accessibility initiative @frankieyan! 

In addition to `aria-label`, what do you think about including an `aria-live="assertive"` attributes along with `role="alert"`?

As it's currently implemented, a customer would have to navigate to the loading indicator to understand that something is loading. Using the attributes above would make the screen reader announce the change without requiring navigation to occur.

Here are some links for references:
* https://codeburst.io/how-to-create-a-simple-css-loading-spinner-make-it-accessible-e5c83c2e464c
* https://www.carbondesignsystem.com/components/loading/accessibility

File: stories/components/LoadingStory.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const LoadingPropTypesChapter = {
 
 const LoadingStory = () => (
     <section className="story loading">
-        <Loading />
+        <Loading aria-label="Loading" />
```
Comment (craigcarlyle) [inline stories/components/LoadingStory.tsx:20]: Could we set "Loading" as the default `aria-label` if one is not provided?

Comment (frankieyan) [reply stories/components/LoadingStory.tsx:20]: 💯

**Comment by gnapse** [comment]
I think that more than an aria label, the loading component should be a role="alert" element. Thoughts?

**Pull request merged**